test/mode/modeVisual.test.ts

```diff
@@ -226,6 +226,22 @@ suite('Mode Visual', () => {
     });
   });
 
+  suite('Various motions in Visual Mode', () => {
```

Comment:
Can we use a more descriptive name? Like `Display line motions` or something?

Reply:
Ok, try that.